src/python/py_image_generation_pipelines.cpp

```diff
@@ -291,6 +293,6 @@ void init_image_generation_pipelines(py::module_& m) {
                 return py::cast(pipe.generate(prompt, params));
             },
             py::arg("prompt"), "Input string",
-            (text2image_generate_docstring + std::string(" \n ")).c_str()
-        );
+            (text2image_generate_docstring + std::string(" \n ")).c_str())
+        .def("decode", &ov::genai::Text2ImagePipeline::decode, py::arg("latent"));;
```

Comment:
is there any plans also demonstrate decode method usage somewhere in readme?

Reply:
https://github.com/openvinotoolkit/openvino.genai/pull/1237